scripts/src/chartprreview/chartprreview.py

```diff
@@ -331,6 +349,7 @@ def main():
     report_generated = os.environ.get("REPORT_GENERATED")
     generated_report_path = os.environ.get("GENERATED_REPORT_PATH")
     generated_report_info_path =  os.environ.get("REPORT_SUMMARY_PATH")
+    provider_delivery = os.environ.get("PROVIDER_DELIVERY")
```

Comment:
I see a lot of `if not ... or ... == "False":` that could be served better by the `environs` Python package, which gives a construct that turns the string value into a boolean:

```python
import environs
from environs import Env

env = Env()
provider_delivery = env.bool("PROVIDER_DELIVERY", False)
```